tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/prose-tests/ClientEncryptionProseTests.cs

```diff
@@ -721,6 +720,217 @@ void TestCase(ClientEncryption testCaseClientEncription, BsonDocument masterKey,
             }
         }
 
+        [SkippableTheory]
+        [MemberData(nameof(DeadlockTest_MemberData))]
+        public void DeadlockTest(
+            string _,
+            int maxPoolSize,
+            bool bypassAutoEncryption,
+            string keyVaultMongoClientKey,
+            int expectedNumberOfClients,
+            string[] clientEncryptedEventsExpectation,
+            string[] clientKeyVaultEventsExpectation,
```

Comment:
If you apply tuples suggestion below:
```suggestion
            (string, string)[] clientEncryptedEventsExpectation,
            (string, string)[] clientKeyVaultEventsExpectation,
```

Reply:
it doesn't really work. In this case, we will have a problem with xunit and shown tests results. NOTE: your suggestion is how I did this originally